test/lit/wasm-split/multi-split.wast

```diff
@@ -7,8 +7,10 @@
 ;; RUN: wasm-dis %t3.wasm | filecheck %s --check-prefix=CHECK-C
 
 (module
- (type $ret-i32 (func (result i32)))
  ;; PRIMARY:      (type $ret-i64 (func (result i64)))
```

Comment:
Why is this not lined up with the declaration below?

Reply:
Because `$ret-i32` is in the way. output $ret-i64 < output $ret-i32 < input $ret-i32 < input $ret-i64.

Reply:
But don't we now match the input order? Or does wasm-split reorder?

Reply:
No, we decided to make all the tools reorder by default and preserve the input order only when --preserve-input-order is passed.

Reply:
Oh, sorry, I remembered that wrong then. Yeah, the issue was the reordered output was more optimal, and we didn't want to risk production builds.

Reply:
Anyhow, using the flag might help here, but if it's just one line sgtm either way.

Reply:
I'll skip it for now because we would have to add --preserve-type-order to each of the 5 commands at the top, which I think would be more annoying than this one line.